### ISSUE TITLE
PEP484 overload

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#595]
In GitLab by @peterdotran on Jun 25, 2017, 20:31

Here is another issue with PEP484

Currently with overload declared functions, the function would need to be declared multiple times. That would result in F811 to raise as an error. Also following the same style from PEP484 and overload, it shows no blank line after each declaration. That would also result in E301 to come up each time.

Example
```python
from typing import overload


@overload
def func(v: int) -> int: ...
@overload
def func(v: float) -> float: ...
@overload
def func(v: str) -> str: ...
def func(v):
    return v
```

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jul 1, 2017, 05:40

F811 is raised by pyflakes as is described in our [contributing](http://flake8.pycqa.org/en/latest/internal/contributing.html#filing-a-bug) and [violation codes](http://flake8.pycqa.org/en/latest/user/error-codes.html#error-violation-codes) sections. As such, you should endeavor to file this bug on [PyFlakes](https://github.com/pycqa/pyflakes).

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Jul 1, 2017, 05:40

closed
